pyccel/codegen/utilities.py

```diff
@@ -67,13 +67,18 @@
 internal_libs["gFTL_functions/Set_extensions"] = (stdlib_path / "gFTL_functions", "gFTL_functions", CompileObj("Set_Extensions.inc",
                                                                      folder="gFTL_functions",
                                                                      has_target_file = False,
+
                                                                      dependencies = (internal_libs['gFTL'][2],)))
```

Comment:
Why this space in the middle of the definition of the `CompileObj`?
Very strange to split here

Reply:
Sorry, fixed!